pipeline_steps/irr_role_tests.groovy

```diff
@@ -20,9 +20,14 @@ def run_irr_tests() {
                   ]
                 ]
               ]) // checkout
-              sh """#!/bin/bash
-              bash ./run_tests.sh
-              """
+              withCredentials(common.get_cloud_creds()){
+                List maas_vars = maas.get_maas_token_and_url()
```

Comment:
This will fail as maas isn't [loaded](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/irr_role_test.yml#L123).